doc/install/install-python.sh

```diff
@@ -139,9 +130,8 @@ else
 	    --enable-unicode=ucs4\
 		--program-suffix=.exe ;
 fi
-
 make 
 make install
-make 
-make install
-echo "..Done"
+echo "Done"
```

Comment:
I think the make and make install was done twice for a reason, but I cant remember what that was??